src/main/java/controllers/TaskFactory.java

```diff
@@ -59,6 +60,7 @@ public ITask createTask(String input) throws DukeException {
 
             } catch (ParseException e) {
                 // Invalid Date and Time, revert back to lazyTiming
+                //throw new InvalidDateTimeException();
```

Comment:
What is this? O.o @sinteary

Reply:
An exception that replaces Parse Error with a more specific message